examples/http-client-browser-pubsub/tests/test.js

```diff
@@ -2,21 +2,23 @@ import { test } from '@playwright/test';
 import { playwright } from 'test-util-ipfs-example'
 import * as ipfsModule from 'ipfs'
 import * as ipfsHttpModule from 'ipfs-http-client'
```

Comment:
Any reason you're still leaving the import here?

Reply:
pinged you on https://github.com/ipfs-examples/js-ipfs-examples/pull/517/files/770401813d56112bd7ecd7701e2dbef58ccd6c77#r1021933665